rotkehlchen/db/history_events.py

```diff
@@ -1244,7 +1245,37 @@ def query_wrap_stats(self, from_ts: Timestamp, to_ts: Timestamp) -> dict[str, An
                 "(SELECT value FROM multisettings WHERE name = 'ignored_asset') GROUP BY chain_id",
                 (from_ts_ms, to_ts_ms),
             )
-            transactions_per_chain = {ChainID.deserialize_from_db(row[0]).name: row[1] for row in cursor}  # noqa: E501
+            transactions_per_chain: dict[str, int] = {}
+            for row in cursor:
+                chain = ChainID.deserialize_from_db(row[0]).to_blockchain()
+                transactions_per_chain[chain.name] = row[1]
+
+            cursor.execute(
+                'SELECT COUNT(DISTINCT history_events.event_identifier) FROM chain_events_info '
+                'JOIN history_events ON chain_events_info.identifier = history_events.identifier '
+                'WHERE history_events.location = ? AND history_events.timestamp >= ? AND history_events.timestamp <= ? '  # noqa: E501
+                "AND history_events.asset NOT IN (SELECT value FROM multisettings WHERE name = 'ignored_asset')",  # noqa: E501
+                (Location.SOLANA.serialize_for_db(), from_ts_ms, to_ts_ms),
+            )
+            if solana_count := cursor.fetchone()[0]:
+                transactions_per_chain[SupportedBlockchain.SOLANA.name] = solana_count
+
+            cursor.execute(
+                'SELECT location, COUNT(DISTINCT event_identifier) as tx_count FROM history_events '  # noqa: E501
```

Comment:
```suggestion
                'SELECT location, COUNT(DISTINCT event_identifier) FROM history_events '  # noqa: E501
```
I don't think `as tx_count` is needed here